src/env.cc

```diff
@@ -1036,6 +1036,21 @@ char* Environment::Reallocate(char* data, size_t old_size, size_t size) {
   return new_data;
 }
 
+void Environment::AddArrayBufferAllocatorToKeepAliveUntilIsolateDispose(
```

Comment:
```suggestion
void Environment::ThisMethodNameIsSoLongItMakesMyEyeballsBleedIDontEven(
```

Reply:
I mean, I’m happy to take a more concise suggestion, but given the very specific use case I’m fine with a long name.